client.c

```diff
@@ -67,19 +73,46 @@ int main(int argc, char **argv)
       switch (opt)
       {
       case 'p':
+         // detect hidden processes
          if (getuid() != 0)
          {
             printf("You must be root to perform this function!\n");
             exit(1);
          }
          checkallquick();
+
+         // Log to LKM
```

Comment:
log first, then run `checkallquick()`, same as -s and -m

Reply:
Placed after the write() part